src/app/modules/admin/components/new-employee/new-employee.component.html

```diff
@@ -1,105 +1,334 @@
+            ></label
+          >
+          <mat-form-field
+            class="mat-form-field__form-control new-employee__form-field"
+            appearance="fill"
+          >
+            <input
+              id="workingSince"
+              matInput
+              formControlName="workingSince"
+              [matDatepicker]="picker_work"
+              placeholder="ДД месяц ГГГГ"
+              required
+            />
+            <mat-datepicker-toggle matSuffix [for]="picker_work">
+              <mat-icon matDatepickerToggleIcon>date_range</mat-icon>
+            </mat-datepicker-toggle>
+            <mat-datepicker #picker_work></mat-datepicker>
+          </mat-form-field>
+        </div>
+        <div class="new-employee__office">
+          <label class="new-employee__label" doText color="gray1" for="office"
+            >Офис<sup
+              class="new-employee__required"
+              doText
+              color="negative_accent"
+              >*</sup
+            ></label
+          >
+          <mat-form-field
+            class="mat-form-field_default-select new-employee__form-field"
+            appearance="fill"
+          >
+            <mat-select
+              id="office"
+              placeholder="Выберите офис"
+              formControlName="office"
+              required
+            >
+              <mat-option *ngFor="let office of offices" [value]="office">{{
+                office
+              }}</mat-option>
+            </mat-select>
+          </mat-form-field>
+        </div>
       </div>
     </div>
     <div class="new-employee__auth">
-      <h2 class="new-employee__heading h3 text_dark_1">Аутентификация</h2>
+      <h3 doText color="dark_1" class="new-employee__heading">
+        Аутентификация
+      </h3>
       <div class="new-employee__inputs">
-        <do-input
-          controlName="email"
-          label="E-mail"
-          required="true"
-          placeholder="Введите E-mail"
-        ></do-input>
-        <div>
-          <div class="new-employee__flex-label">
-            <label
-              class="small_text text-gray new-employee__label"
-              for="password"
-              >Пароль<sup class="new-employee__required">*</sup>
+        <div class="new-employee__form-element">
+          <label class="new-employee__label" doText color="gray1" for="email"
+            >E-mail<sup
+              class="new-employee__required"
+              doText
+              color="negative_accent"
+              >*</sup
+            ></label
+          >
+          <mat-form-field class="new-employee__form-field" appearance="fill">
+            <input
+              id="email"
+              matInput
+              placeholder="Введите E-mail"
+              formControlName="email"
+              required
+            />
+          </mat-form-field>
+        </div>
+        <div class="new-employee__form-element">
+          <div class="new-employee__flex-label new-employee__label">
```

Comment:
Если уж по БЭМ, то нужно делать так:
```suggestion
          <div class="new-employee__label new-employee__label_flex">
```